src/qaqc/01-organize_raw_data.Rmd

```diff
@@ -322,14 +398,24 @@ temperature_archery <- temperature_archery %>%
   mutate(flag = if_else(p1 >= temp_max | p1 <= temp_min, "Out of bounds", flag),
          p1 = if_else(p1 >= temp_max | p1 <= temp_min, NA_real_, p1))
 
+temperature_archery <- temperature_archery %>%
+  mutate(flag = if_else(p1 >= temp_max | p1 <= temp_min,
+    # true
+    ifelse(is.na(flag), "Out of bounds",
+           paste(flag, "Out of bounds", sep = "; ")),
+    # false
+    flag),
+         p1 = if_else(p1 >= temp_max | p1 <= temp_min, NA_real_, p1))
+
+# Here is where I would need to add the paste so that it doesn't replace what is already in the flag column completely
```

Comment:
We could also have different flag columns. Sometimes collapsing into a single flag column is *not* the best way to move forward. 

For example: maybe there is a flag_sonde; flag_temp; flag_do; etc.